### PR TITLE
Add ability to set custom command handler for the 'ClientDriver' (close #1400)

### LINKED ISSUE #1400: Add ability to set custom command handler for the `ClientDriver`

### PR DESCRIPTION
**Pull request by churkin**
@AlexanderMoskovkin @helen-dikareva

### CODE REVIEW

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot /retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

File: src/client/driver/driver.js [hunk 1]
```diff
@@ -412,6 +414,15 @@ export default class Driver {
             .then(() => this._onReady(new DriverStatus({ isCommandResult: true })));
     }
 
+    _onCustomCommand (command) {
+        this.customCommandHandlers[command.type](command).then(response => {
```
Comment (helen-dikareva) [inline src/client/driver/driver.js]: May be declare a variable for `this.customCommandHandlers[command.type]`

File: src/client/driver/driver.js [hunk 2]
```diff
@@ -415,10 +415,13 @@ export default class Driver {
     }
 
     _onCustomCommand (command) {
-        this.customCommandHandlers[command.type](command).then(response => {
+        var handler = this.customCommandHandlers[command.type];
+
+        handler(command).then(result => {
             this._onReady(new DriverStatus({
```
Comment (helen-dikareva) [inline src/client/driver/driver.js]: If you place `new DriverStatus({isCommandResult: true,, result})` in one line linter will not detect wrong indent after pressing `ctrl+alt+L` :)

Comment (churkin) [inline src/client/driver/driver.js]: ok

**Comment by churkin** [comment]
@testcafe-build-bot \retest

File: src/client/driver/driver.js [hunk 3]
```diff
@@ -435,7 +445,9 @@ export default class Driver {
     }
 
     _executeCommand (command) {
-        if (command.type === COMMAND_TYPE.testDone)
+        if (this.customCommandHandlers[command.type])
+            this._onCustomCommand(command);
+        else if (command.type === COMMAND_TYPE.testDone)
```
Comment (inikulin) [inline src/client/driver/driver.js]: Formatting: add new line before `else if`

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
Tests are ok. It's just known problems with Android on Saucelabs

**Pull request merged**